docs/reference/run.md

```diff
@@ -111,6 +111,17 @@ specify to which of the three standard streams (`STDIN`, `STDOUT`,
 
     $ docker run -a stdin -a stdout -i -t ubuntu /bin/bash
 
+If you specify `-i` without STDIN attached, you may encounter unexpected results.  
```

Comment:
Thank you for the contribution @sallyom. Some of the constructions are awkward, some font changes, and tics required:

-->https://gist.github.com/moxiegirl/d3ffe8abdcb4a00b07fb

If you specify `-i` without STDIN attached, you may encounter unexpected results.  
Consider the following example:

    $ docker run -a stderr -a stdout -i --sig-proxy=true ubuntu bash

Any tty-generated signals (CTRL-c, etc) are silently discarded.  If, 
however, `--sig-proxy=false` is set, tty signals are generated and the client 
responds &mdash; not the container.  For this reason, unless you have a use for 
running `-i` without STDIN attached, you should run `-i` with 
`-a stdin` or omit `-a` to attach to STDIN by default.